i18n/config.ts

```diff
@@ -14,6 +14,7 @@ import translationTR from './translations/tr-TR.json';
 import translationUA from './translations/uk-UA.json';
 import translationCN from './translations/zh-CN.json';
 import translatiobTW from './translations/zh-TW.json';
+import translationFI from './translations/fi-FI.json';
```

Comment:
The linting is yelling, the order seems to be important, you might to relocate this one.